test.js

```diff
@@ -1,6 +1,53 @@
+const os = require("os")
+const path = require("path")
 const test = require("ava")
 const minecraftPath = require(".")
 
 test("main", t => {
 	t.is(typeof minecraftPath(), "string")
 })
+
+test("minecraftPath return", t => {
+	const originalPlatform = process.platform
+
+	const testCases = {
```

Comment:
We shouldn't be doing the same computations in the unit tests and the actual code since that would defeat the purpose.

Reply:
That really make sense.